WooCommerce/Classes/ViewRelated/Hub Menu/HubMenu.swift

```diff
@@ -5,17 +5,20 @@ import SwiftUI
 ///
 struct HubMenu: View {
     @ObservedObject private var viewModel: HubMenuViewModel
+    @State private var showWooCommerceAdmin = false
```

Comment:
A minor nit for naming: 
```suggestion
    @State private var showingWooCommerceAdmin = false
```

Reply:
Updated 👍 and I updated also the other properties for showing the other menu.